misc/server_lsm_log.yml

```diff
@@ -0,0 +1,57 @@
+# Generated using inmanta -c /etc/inmanta/inmanta.cfg --log-file /var/log/inmanta/server.log --log-file-level 2 --timed-logs print_default_logging_config server
+formatters:
+  core_console_formatter:
+    # Console formatter with coloring.
+    # This formatter also does indentation of multi-line log message, so all lines are put nicely underneath each other.
+    (): inmanta.logging.MultiLineFormatter
+    fmt: '%(asctime)s %(log_color)s%(name)-25s%(levelname)-8s%(reset)s%(blue)s%(message)s'
+    # Don't shorten logger names by default
+    keep_logger_names: true
+    log_colors:
+      CRITICAL: red
+      DEBUG: cyan
+      ERROR: red
+      INFO: green
+      WARNING: yellow
+    no_color: false
+  # Normal formatter
+  core_log_formatter:
+    format: '%(asctime)s %(levelname)-8s %(name)-10s %(message)s'
+  # Lsm callback
+  lsm_callback_formatter:
+    format: '%(asctime)s %(name)-25s%(levelname)-8s%(message)s'
+handlers:
+  root_handler:
```

Comment:
I would expect `server_handler` similar to `misc/server_log.yml`

Reply:
That is a very good catch, turned out to lead to some more problems. thx